helpdesk/templates/helpdesk/ticket_desc_table.html

```diff
@@ -73,14 +73,37 @@ <h3>{{ ticket.queue.slug }}-{{ ticket.id }}. {{ ticket.title }}</h3>
                         <td>{{ ticket.created|date:"DATETIME_FORMAT" }} ({{ ticket.created|naturaltime }})</td>
                     </tr>
                     <tr>
+                        <!-- show current Assignee for ticket -->
                         <th class="table-active">{% trans "Assigned To" %}</th>
                         <td>
-                            {{ ticket.get_assigned_to }}
-                            {% if _('Unassigned') == ticket.get_assigned_to  %}
-                                <a class="btn btn-primary btn-sm float-right" data-toggle="tooltip" href='?take' title='{% trans "Assign this ticket to " %}{{ request.user.email }}'>
-                                    <i class="fas fa-hand-paper"></i>
-                                </a>
-                            {% endif %}
+                            <!-- assignment drop down -->
+                            <form method="post" action="{% url 'helpdesk:mass_update' %}">
```

Comment:
Using the mass update endpoint is a good idea since we can reuse the logic, but it's important to note that the followup indicating the assignee change will be worded like "Assigned to %(username)s in bulk update".

Reply:
I also noticed that the mass update won't create a TicketChange for assign/unassign like in the update ticket function. I think it's worth fixing it either in this MR or in a new one.